ecommers/lib/core/app_services/product_service.dart

```diff
@@ -12,29 +11,19 @@ class ProductService {
     String searchQuery,
     SortType sortType,
   }) async {
-    final response = await apiService.products(
-      category: category,
-      subCategory: subCategory,
+
+    //TODO: Add category and subCategory type to params
+    return productDataRepository.getProductListByCategory(
+      category: 'Apparel',
```

Comment:
Yeap, if we want to see any data for now. Because when we click on Category we pass null value as category. The repository try to extract data from DB and return null. So for right now to display some data we need to mock it.